django/api/settings.py

```diff
@@ -136,12 +136,12 @@
 
 # Django Rest Framework Settings
 REST_FRAMEWORK = {
-    "DEFAULT_AUTHENTICATION_CLASSES": [
+    "DEFAULT_AUTHENTICATION_CLASSES": [] if "BYPASS_AUTHENTICATION" in os.environ else [
```

Comment:
there needs to be a new middleware for test here, so something like "api.authentication.FakeAuthentication". Maybe the existing file keycloak_authentication.py can be renamed to just authentication with another class added in there or something.

This FakeAuthentication class would then need to return a similar `return user, token` like the other middleware, but the user pulled here can be a static test user pulled from the DB.